app/src/main/java/org/sopt/seminar/presentation/home/HomeActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.sopt.seminar.presentation.home
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import org.sopt.seminar.presentation.follower.FollowerFragment
+import org.sopt.seminar.R
+import org.sopt.seminar.presentation.repo.RepoFragment
+import org.sopt.seminar.User
+import org.sopt.seminar.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+    private var position = FOLLOWER_POSITION
+    private lateinit var binding: ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_home)
+        binding.user = User("김수빈", "23", "ENFJ")
+
+        inintTransactionEvent()
+    }
+
+    private fun inintTransactionEvent() {
+        val followerFragment = FollowerFragment()
+        val repoFragment = RepoFragment()
+
+        supportFragmentManager.beginTransaction().add(R.id.fragment_home, followerFragment).commit()
```

Comment:
메서드별로 라인을 바꿔주면 가독성이 더 좋을 것 같다고 생각합니다. 저는 Android Studio가 제공하는 자동 정렬(Reformat Code)를 애용하는 편인데, 이렇게 하면 제가 짠 코드에 문제가 발생했을 때 제 코드를 독해하기가 더 수월하더군요. :-)
```suggestion
        supportFragmentManager.beginTransaction()
            .add(R.id.fragment_home, followerFragment)
            .commit()
```